src/System.IO.Compression.Brotli/src/Interop/Interop.Brotli.Decoder.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+using size_t = System.IntPtr;
+
+internal static partial class Interop
+{
+    internal static partial class Brotli
+    {
+        internal const string LibNameDecoder = Library.BrotliDecoder;
```

Comment:
Nit: why is this const needed?  It's just to be able to write "LibNameDecoder" rather than "Library.BrotliDecoder" in each DllImport?  We don't do that elsewhere.

Reply:
(And if we do keep it, why internal rather than private?)